Resources/doc/reference/advanced_configuration.rst

```diff
@@ -120,6 +120,35 @@ application's config file:
                     model_manager:          # dependency name, from the table above
                         sonata.order.admin.order.manager  # customised service id
 
+
+Using Annotations to define Admins
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
please extend the documentation for such a big feature, and create a seperate article for this.

all possible annotation needs to be there with some examples, otherwise no one can use this feature

thanks